src/transform-support.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
+function getTransformType(transformPath) {
+  const fs = require('fs-extra');
+
+  if (!fs.existsSync(transformPath)) {
+    throw new Error(`Transform ${transformPath} not found.`);
+  }
```

Comment:
If we use `require.resolve` in `getTransformPath` then I think this can be removed (since `require.resolve` would throw)

Reply:
Done